pkg/ccl/backupccl/backup_planning_tenant.go

```diff
@@ -11,16 +11,33 @@ package backupccl
 import (
 	"context"
 
+	"github.com/cockroachdb/cockroach/pkg/clusterversion"
 	"github.com/cockroachdb/cockroach/pkg/kv"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
 	"github.com/cockroachdb/cockroach/pkg/sql"
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
 	"github.com/cockroachdb/errors"
 )
 
-const tenantMetadataQuery = `
+// tenantMetadataQuery returns a query used to retrieve the tenant metadata.
+//
+// If filterByID is true, the query contains a `WHERE id = $1` clause.
+func tenantMetadataQuery(ctx context.Context, settings *cluster.Settings, filterByID bool) string {
+	query := tenantMetadataQueryWithTenantUsageTable
+	if !settings.Version.IsActive(ctx, clusterversion.TenantUsageTable) {
```

Comment:
We also depend on the `TenantUsageSingleConsumptionColumn`, but the window between these two migrations should be incredibly small I imagine.